src/Controls/tests/TestCases.Shared.Tests/Tests/Issues/Issue16910.cs

```diff
@@ -26,16 +26,17 @@ public void BindingUpdatesFromProgrammaticRefresh()
 		App.Tap("StopRefreshing");
 		App.WaitForElement("IsNotRefreshing");
 	}
-#if TEST_FAILS_ON_CATALYST //Scroll actions cannot be performed on the macOS test server
```

Comment:
Scroll actions are already implemented on macOS but also, now the test use `App.DragCoordinates`.